AirCasting/Graph/Graph.swift

```diff
@@ -203,17 +227,75 @@ struct Graph: UIViewRepresentable {
     func makeUIView(context: Context) -> AirCastingGraph {
         AirCastingGraph()
     }
-
+    
     func updateUIView(_ uiView: AirCastingGraph, context: Context) {
-
+        
         try? uiView.updateWithThreshold(thresholdValues: thresholds.rawThresholdsBinding.wrappedValue)
         
         let entries = stream.allMeasurements?.compactMap({ measurement -> ChartDataEntry? in
             let timeInterval = Double(measurement.time.timeIntervalSince1970)
             let chartDataEntry = ChartDataEntry(x: timeInterval, y: measurement.value)
             return chartDataEntry
         }) ?? []
-        uiView.updateWithEntries(entries: entries, isAutozoomEnabled: isAutozoomEnabled)
+        
+        simplifyGraphline(entries: entries, uiView: uiView)
+        
+        let limitLines = getLimitLines()
+        uiView.updateMidnightLines(with: limitLines)
+    }
+    
+    private func simplifyGraphline(entries: [ChartDataEntry], uiView: AirCastingGraph) {
+        let DEFAULT_LIMIT = 1000
```

Comment:
interesting that you decided to make it snake upper-cased. This is some oldschool `♥️Objective-C♥️` stuff over here ;) But usually this kind of `REALLY_CONSTANT_STUFF` would go to the class scope:
``` Swift
class MegaTurboSuperToolbar: UIToolbar {
   let CONSTANT_STUFF = 999

   ...
}
```

I'm not sure I dislike this tho. @mateuszmackowiak what do you think?

Reply:
Well tbh, I was trying to come up with something but all names were looooooong. So I took a look at this name in Android app and copied the idea. ;D

Reply:
I can move it to class scope :)

Reply:
I think long names are not bad in such cases, we're way passed 80-char wide terminals so we can get a bit more verbose than `default limit` which frankly is not much better than putting a bare `1000` here :) I'd say let's make it `snakeCase`, class scoped and named verbosely like `let simplifiedGraphEntryThreshold = 1000` ?

Reply:
Thanks, I'll do that! :D